service/ildap/user_ildap.go

```diff
@@ -143,7 +143,9 @@ func (x UserService) NewPwd(username string) (string, error) {
 	if username == "admin" {
 		udn = config.Conf.Ldap.AdminDN
 	}
-	modifyPass := ldap.NewPasswordModifyRequest(udn, "", "")
+	//默认产生8位随机密码，不符合公司规范，故生成16位随机密码
+	newPasswd := tools.GenerateRandomPassword()
```

Comment:
在NewPwd函数中，注释说明生成16位随机密码，但实际代码中调用GenerateRandomPassword()函数并未指定长度（默认8位），与注释不符。建议添加一个可指定长度的密码生成函数。

```suggestion
// 默认产生8位随机密码，不符合公司规范，故生成16位随机密码
// newPasswd := tools.GenerateRandomPassword()
newPasswd := tools.GenerateRandomPasswordWithLength(16)